slemicro/delete_vm.sh

```diff
@@ -9,22 +9,29 @@ die(){
 
 # Get the env file
 source ${BASEDIR}/.env
-
+if [ $# -eq 1 ]; then
+    VMNAME=$1
+fi
 VMFOLDER="${VMFOLDER:-~/VMs}"
 
+if [ $(uname -o) == "Darwin" ]; then
 # Stop and delete the VM using UTM
-OUTPUT=$(
-osascript <<END
-tell application "UTM"
-	set vm to virtual machine named "${VMNAME}"
-	if status of vm is started then stop vm
-	repeat
-	  if status of vm is stopped then exit repeat
-  end repeat
-	delete virtual machine named "${VMNAME}"
-end tell
+  OUTPUT=$(
+  osascript <<END
+  tell application "UTM"
+	  set vm to virtual machine named "${VMNAME}"
+	  if status of vm is started then stop vm
+	  repeat
+	    if status of vm is stopped then exit repeat
+    end repeat
+	  delete virtual machine named "${VMNAME}"
+  end tell
 END
-)
+  )
+else
```

Comment:
```suggestion
elif [ $(uname -o) == "GNU/Linux" ]; then
```
For consistency with the other `if`